site/pages/what-we-do/index.js

```diff
@@ -86,3 +76,22 @@ export default function whatWeDo() {
     </div>
   );
 }
+
+export default function whatWeDo() {
+  return (
+    <div>
+      <div className={styles.pageHeaderContainer}>
+        <h1 className={styles.pageHeader}>
```

Comment:
Let's do this in the ally PR rather than this one as this one is unrelated to the what we do header.